test/components/DateRangePicker_spec.jsx

```diff
@@ -699,6 +699,96 @@ describe('DateRangePicker', () => {
     });
   });
 
+  describe('#onStartDateFocus', () => {
+    it('calls props.onFocusChange once', () => {
+      const onFocusChangeStub = sinon.stub();
+      const wrapper = shallow(<DateRangePicker onFocusChange={onFocusChangeStub} />);
+      wrapper.instance().onStartDateFocus();
+      expect(onFocusChangeStub.callCount).to.equal(1);
```

Comment:
fwiw all these would have nicer failure messages if they were like `expect(foo).to.have.property('callCount', 1)` - not a blocker, just a nice-to-have.